pkg/backtest/exchange.go

```diff
@@ -369,14 +369,14 @@ func (e *Exchange) ConsumeKLine(k types.KLine) {
 		matching.klineCache = make(map[types.Interval]types.KLine)
 	}
 
-	kline1m, ok := matching.klineCache[k.Interval]
+	requiredKline, ok := matching.klineCache[k.Interval]
 	if ok { // pop out all the old
-		if kline1m.Interval != types.Interval1m {
-			panic("expect 1m kline, got " + kline1m.Interval.String())
+		if requiredKline.Interval.Seconds() < requiredInterval.Seconds() {
```

Comment:
use `!=` here.
The order of kline is already restricted by SQL, so the first popped out kline should be the same as the requiredInterval

Reply:
Aalready tried. Not working

Reply:
should re-sync databse